sdks/python/apache_beam/examples/game_stats_it_test.py

```diff
@@ -0,0 +1,153 @@
+#
```

Comment:
Should this file be moved to https://github.com/apache/beam/tree/master/sdks/python/apache_beam/examples/complete/game, so that it can be co-located with the code it is testing?

Reply:
It makes sense, I have also moved the other IT tests.